src/main/java/com/dreammaster/gthandler/recipes/ImplosionCompressorRecipes.java

```diff
@@ -23,318 +27,426 @@ public class ImplosionCompressorRecipes implements Runnable {
+                    .itemOutputs(
+                            CustomItemList.IronDualCompressedPlates.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Ash, 2L))
+                    .noFluidInputs().noFluidOutputs().duration(20 * TICKS).eut(TierEU.RECIPE_LV)
+                    .metadata(ADDITIVE_AMOUNT, 2).addTo(sImplosionRecipes);
+
+            GT_Values.RA.stdBuilder()
+                    .itemInputs(GT_ModHandler.getModItem(GalacticraftCore.ID, "item.meteoricIronIngot", 2L, 1))
+                    .itemOutputs(
+                            CustomItemList.MeteoricIronDualCompressedPlates.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Ash, 2L))
+                    .noFluidInputs().noFluidOutputs().duration(20 * TICKS).eut(TierEU.RECIPE_LV)
+                    .metadata(ADDITIVE_AMOUNT, 2).addTo(sImplosionRecipes);
+
         }
         if (!BartWorks.isModLoaded()) {
-            GT_Values.RA.addImplosionRecipe(
-                    CustomItemList.HeavyDutyAlloyIngotT4.get(1L),
-                    32,
-                    CustomItemList.HeavyDutyPlateTier4.get(1L),
-                    GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Osmiridium, 4L));
+
+            GT_Values.RA.stdBuilder().itemInputs(CustomItemList.HeavyDutyAlloyIngotT4.get(1L))
+                    .itemOutputs(
+                            CustomItemList.HeavyDutyPlateTier4.get(1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Osmiridium, 4L))
+                    .noFluidInputs().noFluidOutputs().duration(20 * TICKS).eut(TierEU.RECIPE_LV)
+                    .metadata(ADDITIVE_AMOUNT, 32).addTo(sImplosionRecipes);
+
         }
-        GT_Values.RA.addImplosionRecipe(
-                CustomItemList.HeavyDutyAlloyIngotT5.get(1L),
-                40,
-                CustomItemList.HeavyDutyPlateTier5.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Europium, 5L));
-        GT_Values.RA.addImplosionRecipe(
-                CustomItemList.HeavyDutyAlloyIngotT6.get(1L),
-                48,
-                CustomItemList.HeavyDutyPlateTier6.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Tritanium, 6L));
-        GT_Values.RA.addImplosionRecipe(
-                CustomItemList.HeavyDutyAlloyIngotT7.get(1L),
-                56,
-                CustomItemList.HeavyDutyPlateTier7.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Neutronium, 7L));
-        GT_Values.RA.addImplosionRecipe(
-                CustomItemList.HeavyDutyAlloyIngotT8.get(1L),
-                64,
-                CustomItemList.HeavyDutyPlateTier8.get(1L),
-                GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.BlackPlutonium, 8L));
+
+        GT_Values.RA.stdBuilder().itemInputs(CustomItemList.HeavyDutyAlloyIngotT5.get(1L))
```

Comment:
tier 4 is fine for some reason, but tier5-8 do not appear ingame

Reply:
tier 4 is handled in BW it seems, at least not in the ImplosionCompressorRecipes.java